specifyweb/frontend/js_src/lib/components/Merging/index.tsx

```diff
@@ -255,7 +268,7 @@ function Merging({
                 errorMode: 'dismissible',
               }
             ).then((response) => {
-              setMergeId(response.data);
+              setMergeId(JSON.parse(response.data));
```

Comment:
don't do JSON.parse here
instead just change this:
```
	                headers: {
                  Accept: 'text/plain',
                },
```
to this

```
	                headers: {
                  Accept: 'application/json',
                },
```

this way ajax. will take care of parsing, and it will also ensure that back-end returned valid JSON response (and if not, take care of displaying proper error to the user)